modules/sdk-core/src/bitgo/utils/opengpgUtils.ts

```diff
@@ -156,34 +157,53 @@ export async function createSharedDataProof(
 }
 
 /**
- * Creates an Eddsa KeyShare Proof by appending an ed25519 subkey to an armored gpg private key.
+ * Creates a KeyShare Proof based on given algo.
+ *
+ * Creates an EdDSA KeyShare Proof by appending an ed25519 subkey to an armored gpg private key.
+ * Creates an EcDSA KeyShare Proof by Append a secp256k1 subkey (auth) to a PGP keychain.
+ *
+ * For EdDSA:
+ * uValue itself is used to construct by EDDSA signature
```

Comment:
this is a confusing section (also misspelled signature)

in both cases (ECDSA and EdDSA), the public form of the u-value is the public params of the auth subkey.   It *IS* is the public subkey.  The proof itself, is really the PGP signature/certification of these subkeys (the fact that ecdsa is used underneath, is really an implementation detail of PGP and probably not worth mentioning as that may be confusing to those using this function and its `algo` param)

Reply:
ok, i will remove this section